src/components/quiz/Quiz.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+import {Field, reduxForm} from 'redux-form';
+import {connect} from 'react-redux';
+
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+import QuestionsList from './QuestionsList';
+
+const questions = [
```

Comment:
You can start utilising mock REST server from this repo. You should add the data to `db.json`, and run `npm run api`